.github/workflows/docs_build_and_deploy.yml

```diff
@@ -27,7 +27,9 @@ jobs:
     needs: linting
     runs-on: ubuntu-latest
     steps:
-      - uses: neuroinformatics-unit/actions/build_sphinx_docs@v2
+      - uses: neuroinformatics-unit/actions/build_sphinx_docs@main
+        with:
+          python-version: 3.11
```

Comment:
Any chance 3.12 works? Just gives us slightly more runway before it needs changing.